src/main/python/__cvd_risk_bmi.py

```diff
@@ -91,17 +94,32 @@ def heart_age_by_cvd_bmi(cvd,age,sex):
     else:
         cvd = float(cvd)
     if (sex == 'Male'):
-        X = BETA_MEN[1]*np.log(22) - CONST_MEN
+        X = BETA_MEN[1]*np.log(22) - CONST_MEN + CONST_MEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_MEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_MEN[0])))
         return np.round(result)
     else
-        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN
+        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN + CONST_WOMEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_WOMEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_WOMEN[0])))
         return np.round(result)
 
+def validate_heart_age(cvd,age,sex):
+    #handle the condition wheere cvd is None
+    if cvd is None:
+        return 0
+    if str(cvd) == n_a:
+        return 0
+    else:
+       split_gender = sex.split(",")
+       return 1
+
+def add_heart_age_adjuster(heart_age):
+    heart_age_adjuster : int = "1"
+    
+    return heart_age + heart_age_adjuster
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/adhirpotdarbito/pranahealth/pull/3#issuecomment-2303873311>#bbb5f2</a><span> - 08/22/2024, 06:23 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Attempting to add a string value ("1") to heart_age, which is likely a numeric type.
</div>
<div id="fix">
<b>Fix:</b> Convert heart_age_adjuster to an integer before adding it to heart_age.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -120,3 +120,3 @@
 -    heart_age_adjuster : int = "1"
 -
 -    return heart_age + heart_age_adjuster
 +    heart_age_adjuster: int = 1
 +
 +    return heart_age + heart_age_adjuster
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged